catalog/spatial/registry/registry-app/src/main/resources/features.xml

```diff
@@ -28,7 +28,7 @@
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
 
-    <feature name="registry-app" install="manual" version="${project.version}"
+    <feature name="registry-app" install="auto" version="${project.version}"
```

Comment:
This is good for testing, but have we decided that we're going to keep the registry-app on by default?